packages/server/postgres/utils/checkTeamEq.ts

```diff
@@ -1,17 +1,18 @@
-import Team from '../../database/types/Team'
 import getRethink from '../../database/rethinkDriver'
-import {checkTableEq} from './checkEqBase'
+import Team from '../../database/types/Team'
 import getTeamsByIds from '../queries/getTeamsByIds'
+import {checkTableEq} from './checkEqBase'
 
-const alwaysDefinedFields: (keyof Partial<Team>)[] = [
+const alwaysDefinedFields: (keyof Team)[] = [
   'name',
   'createdAt',
   'isArchived',
   'isPaid',
   'tier',
   'orgId',
-  'updatedAt'
+  'isOnboardTeam'
 ]
+const ignoredFields: (keyof Team)[] = ['updatedAt', 'createdBy']
```

Comment:
Instead of ignoring it, I think we should put it in the `maybeUndefinedFieldsDefaultValues` field in the following lines. See reference on [checkUserEq](https://github.com/ParabolInc/parabol/blob/09c12a5267f0a04fe7370ac9a63f5b4c879256c9/packages/server/postgres/utils/checkUserEq.ts#L19-L34)

Reply:
Yep, you are right!